.gitignore

```diff
@@ -0,0 +1 @@
+node_modules/
```

Comment:
Adding .node_modules to a gitignore will keep the node_modules and all the npm package data out of git